packages/rest/src/rest-server.ts

```diff
@@ -515,6 +520,23 @@ export class RestServer extends Context implements Server {
     this.sequence(SequenceFromFunction);
   }
 
+  private _addServerSpec(opts: ServerOptions) {
+    const spec = this.getSync(RestBindings.API_SPEC);
+    spec.servers = [
+      createServerAsUrl({
+        protocal: 'http',
```

Comment:
Typo, should be `protocol`?

Can we leave out protocol, hostname and port from the server URL please? AFAICT from the examples found elsewhere in your pull request, it's possible to specify server url as a path only. The semantics is same as in Swagger v2, meaning the clients should use the same protocol, hostname and port as they used to obtain the spec.

See https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#server-object
> This URL supports Server Variables and MAY be relative, to indicate that the host location is relative to the location where the OpenAPI document is being served. 

I.e. when `https://example.com:8080/openapi.json` says `{server: {url: "/api"}}`, it's telling the client to find the API at `https://example.com:8080/api`.

This is important because in a typical production setting, LoopBack applications are sitting behind a gateway/reverse proxy and don't know their public URL. For example, the LoopBack app may think it's listening on `http://192.168.100.1:80`, but the clients are accessing it at `https://api.bluecorp.com`.